mkosi/kmod.py

```diff
@@ -179,12 +188,10 @@ def gen_required_kernel_modules(
     if (root / "usr/lib/firmware").exists():
         yield root / "usr/lib/firmware"
```

Comment:
This yield and the three above could also be removed by adjusting the path below to just `root / "usr/lib"`